go.mod

```diff
@@ -107,6 +107,7 @@ require (
 	github.com/Masterminds/goutils v1.1.1 // indirect
 	github.com/Masterminds/sprig/v3 v3.2.2 // indirect
 	github.com/Masterminds/squirrel v1.5.2 // indirect
+	github.com/Microsoft/go-winio v0.5.1 // indirect
```

Comment:
Where has this come from?

Reply:
```
❯ go mod why github.com/Microsoft/go-winio
# github.com/Microsoft/go-winioft/go-winio
github.com/smartcontractkit/chainlink/core/scripts/chaincli/handler
github.com/docker/docker/client
github.com/docker/go-connections/sockets
github.com/Microsoft/go-winio
```

Reply:
Right, docker client requires this package